### DISCUSSION TITLE
Dichvucoder chat community.

### DISCUSSION THREAD

**Discussion by Dichvucoder** [Dichvucoder/Dichvucoder-9#1] [Announcements]
Hi, welcome to the Dichvucoder chat community

**Comment by salpikaw** [comment]
hello... is there a way to install dichvucoder to debian or kali-linux wsl?

**Reply by Dichvucoder** [reply]
Yes, go to the issues of creating a request with your operating system, we will create dichvucoder extension for that operating system in no time.

**Reply by salpikaw** [reply]
thanks, request made
